tox.ini

```diff
@@ -1,8 +1,9 @@
 [tox]
 envlist =
+    lint
     py27,py34,py35,py36,py37,py38,pypy,pypy3,
     docs,
-    {py2,py3}-cover,coverage
+    py36-cover,coverage
```

Comment:
This goes along with my previous comment, we NEED to run on both python 2 and python 3 to get full coverage results.

Please revert this change.